va_explorer/templatetags/va_explorer_tags.py

```diff
@@ -11,6 +12,27 @@
 
 register = template.Library()
 
+@register.filter
+def replace(value):
+    value = value.strip()
+    if("dk" == value or "DK" == value):
+        return "Don't know"
+    elif("nan" == value):
+        return "n/a"
+    elif("veryl" == value):
+        return "Very Low"
+    elif("ref" == value):
+        return "refuse to answer"
+    else:
+        return value
+
+@register.filter
+def upper(value):
+    return value.upper()
+
+@register.filter
+def capitalize_every_first_letter(value):
+    return ' '.join(s[:1].upper() + s[1:] for s in value.split(' '))
 
```

Comment:
Are these filters used somewhere?

Reply:
They're not, I liked the idea of including some filter functions to serve as an example of how they are written, or also in the case that we want to come back to use them again.

Also, per .title(), it doesn't work for cases of strings with apostrophe `'`, such as somebody mentioned in [this thread](https://stackoverflow.com/questions/8199966/python-title-with-apostrophes)


I can take these functions out if preferred

Reply:
I see. I think this custom method needs to be tested a bit more and revised. 

E.g., for string "my cAT", the custom method returns 'My CAT', which I don't think is quite correct. I think the intention is to use titlecase and return 'My Cat'. 

"My cAT".title() returns 'My Cat'. 

If you want to leave `capitalize_every_first_letter` in the code after debugging it, that seems fine. I don't think it's worth having a filter for upper, since it's just wrapping an existing Python function.

Reply:
I'll just remove the two functions from the PR since I think they're not used anyway. 
I intended on using them at first but later decided not to but still kept them in because I thought they were helpful to look at when writing new filters.